gateway/api/migrations/0011_jobconfig_job_config.py

```diff
@@ -6,7 +6,6 @@
 
 
 class Migration(migrations.Migration):
-
```

Comment:
Kind of annoying to have to update these files (migrations 0011, 0012) in this PR, but the black version being resolved for the gateway wants this fixed :)